libfreerdp/core/nego.h

```diff
@@ -150,6 +151,7 @@ FREERDP_LOCAL void nego_set_restricted_admin_mode_required(rdpNego* nego,
 FREERDP_LOCAL void nego_set_gateway_enabled(rdpNego* nego, BOOL GatewayEnabled);
 FREERDP_LOCAL void nego_set_gateway_bypass_local(rdpNego* nego,
         BOOL GatewayBypassLocal);
+FREERDP_LOCAL void nego_set_gateway_access_token(rdpNego* nego, char* token);
```

Comment:
should be `const char* token`

Reply:
Removed change. Was not needed, sorry for that.

Reply:
You still have to remove this from the header